crowbar_framework/app/models/service_object.rb

```diff
@@ -1551,26 +1544,6 @@ def run_remote_chef_client(node, command, logfile_name)
     }
   end
 
-  def chef_daemon(action, node_list)
```

Comment:
@MaximilianMeister That approach leads to a gradual accumulation of unused dead code, so it's not a good idea.  If we need it later we can easily retrieve from git.  But anyway this change is on a stable branch so it's only going to see bugfixes, so we probably won't ever need it again here.